src/ontoweaver/base.py

```diff
@@ -508,6 +504,17 @@ def __repr__(self):
 
         return f"<Transformer:{type(self).__name__}({params}) {','.join(columns)}{link}>"
 
+    def create(self, item):
+
+        try:
+            res = str(item)
+            if self.output_validator(pd.DataFrame([res], columns=["cell_value"])):
+                yield res
```

Comment:
We probably don't want to make a generator here, since there is no loop (edit: see below for the problems it produced).
```suggestion
                return res
```